package.json

```diff
@@ -30,24 +30,26 @@
     "lib"
   ],
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
   },
   "devDependencies": {
     "@commitlint/cli": "^16.0.3",
     "@commitlint/config-conventional": "^16.0.0",
     "@ossjs/release": "^0.2.1",
-    "@types/jest": "^26.0.15",
-    "@types/set-cookie-parser": "^2.4.0",
+    "@types/jest": "^29.5.7",
+    "@types/set-cookie-parser": "^2.4.5",
     "commitizen": "^4.2.4",
     "cz-conventional-changelog": "^3.3.0",
-    "jest": "^26.6.3",
-    "rimraf": "^3.0.2",
-    "set-cookie-parser": "^2.4.6",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-environment-miniflare": "^2.14.1",
```

Comment:
I mistakenly through there might have been a version mismatch due to the old version of jest (causing the ci failure), but this was not the case.  I had already updated versions here, but can roll the back if you prefer

Reply:
If the tests pass, let them be.